assets/js/index.js

```diff
@@ -0,0 +1,310 @@
+// get screens
+const header = document.getElementById("header");
+const titleScreen = document.getElementById("title-screen");
+const questionScreen = document.getElementById("question-screen");
+const gameOverScreen = document.getElementById("game-over-screen");
+const highScoreScreen = document.getElementById("high-score-screen");
+const highScoreError = document.getElementById("high-score-error");
+const timerItem = document.getElementById("timer");
+
+// target buttons
+const startBtn = document.getElementById("start-btn");
+const initalsForm = document.getElementById("initials-form");
+const homeBtn = document.getElementById("home-btn");
+const clearBtn = document.getElementById("clear-btn");
+const viewScoresBtn = document.getElementById("view-scores-btn");
+
+// variables
+const timerAmount = 60;
+const timePenalty = 10;
+const scoreAmount = 100;
+
+// game constants
+let questionNumber = 0;
+let score = 0;
+let highScoreArray = JSON.parse(localStorage.getItem("highScores")) || [];
+let timer = timerAmount;
+let timeRemaining = true;
+let timing = null;
+let timerStarted = false;
+let questions = [];
+
+// get questions from API
+const getAPIQuestions = () => {
+  startBtn.classList.add("hide");
+  fetch("https://opentdb.com/api.php?amount=10&difficulty=easy&type=multiple")
+    .then((result) => {
+      return result.json();
+    })
+    .then((loadedQuestions) => {
+      questions = loadedQuestions.results.map((loadedQuestions) => {
+        const formattedQuestions = {
+          question: loadedQuestions.question,
+          answers: [
+            { text: loadedQuestions.correct_answer, correct: true },
+            { text: loadedQuestions.incorrect_answers[0], correct: false },
+            { text: loadedQuestions.incorrect_answers[1], correct: false },
+            { text: loadedQuestions.incorrect_answers[2], correct: false },
+          ],
+        };
+        shuffleAnswers(formattedQuestions.answers);
+        formattedQuestions.question = formatText(formattedQuestions.question);
+        return formattedQuestions;
+      });
+      startQuiz();
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+};
+
+// randomise answers
+const shuffleAnswers = (answersArray) => {
+  for (let i = 0; i < answersArray.length; i++) {
+    answersArray[i].text = formatText(answersArray[i].text);
+    let currentIndex = answersArray.length,
```

Comment:
We shouldn't be declaring variables like this. If you want to declare `randomIndex` as an empty variable here we should see:

`let currentIndex = answersArray.length`
`let randomIndex`